### ISSUE TITLE
Jumboji renders in black and white in Ubuntu 17.10

### ISSUE THREAD

**Issue by kurtwheeler** [richard-clark/jumboji#1]
![screenshot from 2018-02-08 09-55-31](https://user-images.githubusercontent.com/4026833/35979459-68c2c03a-0cb6-11e8-8379-b5243f7d40ec.png)

I'm not actually sure if it's related to Ubuntu or not, but the colors seem to work for @ramenhog running Chrome on MacOS.

**Comment by richard-clark** [comment]
Right now, Jumboji will use whatever emoji display natively in your OS. It looks like Ubuntu 17 doesn't yet support colored emoji ([Ubuntu 18 will](http://www.omgubuntu.co.uk/2017/11/ubuntu-finally-long-last-time-support-color-emoji)), so Jumboji uses black-and-white emoji instead.

I think being able to specify a particular icon style (say macOS) would definitely be a nice-to-have though.

**Comment by kurtwheeler** [comment]
Aww bummer :(

Ya gotta be careful about specifying icon style though. I think that's maybe how Slack ran into their emoji woes.

**Comment by richard-clark** [comment]
Yeah, true. Maybe an open-source emoji font... It's a shame though, because I think the Apple emoji are the typically the most well-known...
